tools/releasing/deploy_staging_jars.sh

```diff
@@ -41,7 +41,11 @@ fi
 
 cd ${PROJECT_ROOT}
 
-echo "Deploying to repository.apache.org"
-${MVN} clean deploy -Papache-release -DskipTests -DretryFailedDeploymentCount=10 $CUSTOM_OPTIONS
+FLINK_VERSIONS=("1.15" "1.16" "1.17")
```

Comment:
We probably don't want to modify this script every time we prepare a new Flink ML release.

Would it be simpler to use environment variables to pass versions and document the usages of this environment variables in the release wiki?